lib/config/typescript-react.js

```diff
@@ -1,11 +1,4 @@
-const merge = require('merge');
-
 module.exports = {
-  extends: [
-    // react should come before typescript
-    'plugin:shopify/react',
-    'plugin:shopify/typescript',
-  ],
-
-  rules: merge(require('./rules/typescript-react')),
+  // DEPRECATED: plugin:shopify/react is sufficient now
```

Comment:
`DEPRECATED: Please use plugin:shopify/react. plugin:shopify/typescript-react shall be removed in a future major release` to lay out a future plan perhaps?